boa/integrations/jupyter/browser.py

```diff
@@ -39,26 +37,77 @@
 nest_asyncio.apply()
 
 
-if not colab_eval_js:
-    # colab creates a new iframe for every call, we need to re-inject it every time
-    # for jupyterlab we only need to do it once
-    install_jupyter_javascript_triggers()
+def _install_javascript_triggers():
+    """Run the ethers and titanoboa_jupyterlab Javascript snippets in the browser."""
+    cur_dir = dirname(realpath(__file__))
+    with open(join(cur_dir, "jupyter.js")) as f:
+        jupyter_js = f.read()
+    prefix = os.getenv("JUPYTERHUB_SERVICE_PREFIX", "..")
+    js = jupyter_js.replace("$$JUPYTERHUB_SERVICE_PREFIX", prefix)
```

Comment:
note potential for merge conflict with https://github.com/vyperlang/titanoboa/pull/242

Reply:
I'll fix conflits as they come